docs/UserGuide.md

```diff
@@ -330,3 +366,14 @@ Displaying statistics of lower price and higher price of preferences of clients
 | Favourites window | fw                                                                                                     | -                                                                                                                                    |
 | Statistics window | stats                                                                                                  | -                                                                                                                                    |
 | Reminder window   | rm                                                                                                     | -                                                                                                                                    |
+
+#Keyboard Shortcuts
+
+| Key | Function                |
+|-----|-------------------------|
+| F1  | Opens Help Window       |
+| F2  | Opens stats Window      |
+| F3  | Opens favourties Window |
+| F4  | Opens reminders Window  |
```

Comment:
`stats` -> rename as `Statistics` 
typo for `favourties` -> rename as `Favourites` 
`reminders` -> rename as `Reminders` 

renaming of windows to match the title of the respective windows